src/foundry/client/data/documents/macro.d.mts

```diff
@@ -34,10 +34,9 @@ declare global {
      * Execute the Macro command.
      * @param scope - Macro execution scope which is passed to script macros
      * @returns A created ChatMessage from chat macros or returned value from script macros
+     * @remarks Does not actually return the chat message
      */
```

Comment:
I ended up [filing a ticket immediately](https://github.com/foundryvtt/foundryvtt/issues/11505) because it seems to be less of a docs issue and more of a runtime issue.